scripts/test.sh

```diff
@@ -1,12 +1,11 @@
-#!/usr/bin/env bash
```

Comment:
it looks like we can remove this file now as we have the test scripts under scripts/ doing the same ..

Reply:
here all tests are run, and i like to run it before I do PRs etc. Unless you have a strong objection i'd like to keep it.